content/publications.md

```diff
@@ -28,6 +28,12 @@ newsletters:
 ---
 Informing the public about our data, our findings and what the data means is of the utmost importance to the Texas Justice Initiative. More information about our data sets can be found [here](https://texasjusticeinitiative.org/about-the-data/), and all of our data is available through data.world (free account required) <a href="https://data.world/tji" target="_blank" rel="noopener noreferrer">here</a>. Below, you’ll find the Texas Justice Initiative’s latest press releases and reports.
 
+## Mapping COVID deaths in Custody
+
+An interactive map updated monthly with data tracking COVID related deaths of incarcerated individuals within the state of Texas. 
+
+[Mapping COVID deaths in Custody](localhost:3333/publications/covid-deaths-in-texas)
```

Comment:
We'll need to update this link before merging.

That makes me think... what do you think about merging this PR without this change to the publications page? We'd have the map in production and share the URL internally. After we've allowed folks to poke at it and provide feedback, we could add the link to the publications page, making it "live" to the public.

Reply:
Yes, I think that's a very good idea, in that case we don't need to touch this file at all until it's all ready. I'm not sure how to remove this change as a part of the review.

Reply:
Awesome! If you make commits to your branch locally and then push them up to GitHub, those changes will be reflected in this PR.